src/devtools/client/debugger/src/actions/breakpoints/index.js

```diff
@@ -184,7 +185,13 @@ export function addBreakpointAtLine(cx, line, shouldLog = false, disabled = fals
     };
 
     const options = {};
-    const file = source.url.split("/").pop();
+    let file = source.url?.split("/").pop();
+
+    if (!file) {
+      const { host } = getSourceNames(source.url || source.relativeUrl);
+      file = host;
+    }
+
```

Comment:
we should probably move this code into a logValue helper so it's not duplicated in 2 places